bin/build.sh

```diff
@@ -60,7 +60,7 @@ CONFIGURE_INSTALL_PREFIX=
 CMAKE_INSTALL_PREFIX=
 
 # Partial list of dependencies; the rest are added depending on the platform
-DEPENDENCIES="git cmake python3-yaml python3-psutil unzip wget ninja-build"
+DEPENDENCIES="git cmake python3-yaml python3-psutil unzip wget ninja-build libboost-all-dev subversion"
```

Comment:
Why does it require subversion? That is really weird.

Reply:
This is something we have to figure out for building sea-dsa.

Reply:
It's no longer needed.